Transcelerator/TxlInfo.cs

```diff
@@ -65,38 +65,67 @@ public TxlInfo()
 			object[] attributes = assembly.GetCustomAttributes(typeof (AssemblyCopyrightAttribute), false);
 			if (attributes.Length > 0)
 				m_copyright = ((AssemblyCopyrightAttribute) attributes[0]).Copyright;
+
+			m_showDetailsLabel = m_btnDetails.Text;
+		}
+
+		private string VersionInfo => Format(m_lblAppVersion.Text, m_versionStr);
+
+		private string BuildDateInfo => Format(m_lblBuildDate.Text, m_buildDate);
+
+		private string FullCopyrightNotice
+		{
+			get
+			{
+				if (m_copyright == null)
+					m_copyright = "© SIL Global";
+				return Format(m_lblCopyrightAndLicense.Text, m_copyright);

```

Comment:
The interaction of this property with the ``m_copyright`` variable is a little strange, but I'm not sure of a better way to do it. Maybe you could load the HTML file and extract the copyright even if the WebView fails to load? Probably not worth the effort.